### PR TITLE
Is pin and is close feature changes #MN-2

### PR DESCRIPTION
**Pull request by miteshqodic**
Is pin and is close feature changes #MN-2

### CODE REVIEW

File: public/js/user-thread.js
Comment (illuminate3) [inline public/js/user-thread.js:1]: Could you move this to asset/js and then add to webpack?

Can you move the inline styles into a sass style sheet and add then have the app.scss import the file?

File: templates/thread/index.html.twig
Comment (illuminate3) [inline templates/thread/index.html.twig:1]: sweet alert should be added to package.json and then have webpack call it.
Jquery is already in the webpack - "jquery": "^3.6.0", is available.

As above, can we try to move the CSS to app.scss or an extra file?

**Review by illuminate3 — CHANGES_REQUESTED** [review]
Code is looking great!

Just need to get more into a Symfony way of doing things.

File: public/js/user-thread.js
Comment (miteshqodic) [reply public/js/user-thread.js:1]: > Could you move this to asset/js and then add to webpack?
> 
> Can you move the inline styles into a sass style sheet and add then have the app.scss import the file?

Ok noted, will make suggested changes

File: templates/thread/index.html.twig
Comment (miteshqodic) [reply templates/thread/index.html.twig:1]: > sweet alert should be added to package.json and then have webpack call it. Jquery is already in the webpack - "jquery": "^3.6.0", is available.
> 
> As above, can we try to move the CSS to app.scss or an extra file?

Ok noted, will make suggested changes

**Pull request merged**